pom.xml

```diff
@@ -305,24 +305,23 @@
             </exclusions>
         </dependency>
 
-        <!-- discard all logging from hadoop -->
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>jcl-over-slf4j</artifactId>
+            <artifactId>slf4j-api</artifactId>
             <version>${dep.slf4j.version}</version>
-            <scope>runtime</scope>
         </dependency>
 
+        <!-- discard all logging from hadoop -->
```

Comment:
This comment seems incorrect now, since we're not discarding output. That's a big change (from a behavioral standpoint) and should be a separate commit. We'll need to tell users about this as they will likely want to change their log level configuration.